replay/utils.py

```diff
@@ -847,3 +848,36 @@ def return_recs(
 
     recs.write.parquet(path=recs_file_path, mode="overwrite")
     return None
+
+
+def save_picklable_to_parquet(obj: Any, path: str) -> None:
+    """
+    Function dumps object to disk or hdfs in parquet format.
+
+    Args:
+        obj: object to be saved
+        path: path to dump
+    """
+    sc = SparkSession.getActiveSession().sparkContext
```

Comment:
why it is not State().session?  + line 880

Reply:
> why it is not State().session? + line 880

For no particular reason, just getting the session without additional abstractions. should be changed to `State().session`?
Line 880 the same.